docs/sources/release-notes/v2.3.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Grafana Mimir version 2.3 release notes"
+menuTitle: "V2.3 release notes"
+description: "Release notes for Grafana Mimir version 2.3"
+weight: 300
+---
+
+# Grafana Mimir version 2.3 release notes
+
+Grafana Labs is excited to announce version 2.3 of Grafana Mimir, the most scalable, most performant open source time series database in the world.
+
+The highlights that follow include the top features, enhancements, and bugfixes in this release. If you are upgrading from Grafana Mimir 2.2, there is upgrade-related information as well.
+For the complete list of changes, see the [Changelog](https://github.com/grafana/mimir/blob/main/CHANGELOG.md).
+
+## Features and enhancements
```

Comment:
I think the following are worth a note:
- * [FEATURE] Compactor: Adds the ability to delete partial blocks after a configurable delay. This option can be configured per tenant. #2285
- * [FEATURE] Querier: Added support for tenant federation to metric metadata endpoint. #2467

Reply:
Marco, what is the value of the first feature? Why would I want to delete partial blocks? Saves me more disk space? Are these partial blocks just bad blocks that were hanging around previously and never got cleaned up?

Reply:
I think the second one is nice! 
@56quarters @pracucci maybe we could also update
https://grafana.com/docs/mimir/latest/operators-guide/securing/authentication-and-authorization/
to specify which endpoints do and don't support tenant federation. 

Today we have:

```You can federate queries across multiple tenants by using true in -tenant-federation.enabled=true. When you specify tenant IDs, separate them with a pipe (|) character in the ‘X-Scope-OrgID’ header, as in the example X-Scope-OrgID: tenant-1|tenant-2|tenant-3.
```

Ideally we'd specify this on our HTTP API docs instead of that page I mentioned above, labeling the APIs that support federation in the same way we label which APIs require the use of authentication. But that perhaps is a higher effort ask.

Reply:
> Are these partial blocks just bad blocks that were hanging around previously and never got cleaned up?

This one. Compactor will complain about them forever, unless you manually do something (Mimir correctness is not affected). With the new feature, the compactor will auto-cleanup them.

Reply:
> @56quarters @pracucci maybe we could also update
https://grafana.com/docs/mimir/latest/operators-guide/securing/authentication-and-authorization/
to specify which endpoints do and don't support tenant federation.

Good point. Updating the auth doc page LGTM, but I think longer term we should have a dedicated page for the query federation.

Reply:
@pracucci re: partial block cleanup - is there a reason that it is off by default? Just because its experimental still? This seems like something that any user would want to turn on right? Since it would serve to cleanup these partial blocks that are corrupted/useless/leftover from some aborted compaction?

Reply:
> Why would you want to ever keep those partial blocks around if they're useless?

It was a new feature, so we didn't enable it by default before testing on our servers.

Partial blocks can be a result of compactor interrupted during upload, or unfinished backfill upload. If user is not experiencing any of this, they don't need the feature.

I think having a default value for example `24h` would be OK.

Reply:
> I think having a default value for example 24h would be OK.

We're still rolling out at Grafana Labs. I would wait next release before enabling it by default, but agree we should.